src/main/java/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -107,6 +110,8 @@ public DriveSubsystem(PIDManagerFactory pidFactory, XPropertyManager propManager
         this.minTranslateSpeed = pf.createPersistentProperty("Minimum translate speed", 0.02);
         this.minRotationalSpeed = pf.createPersistentProperty("Minimum rotational speed", 0.02);
 
+        this.driverOrientedDriveEnabled = pf.createPersistentProperty("Enable Driver Oriented Drive", true);
```

Comment:
❓  Should this be FieldOrientedDrive instead? I kind of initially assumed based on the current name the exact opposite behavior 💫

Reply:
alternative idea: `allianceDriveStationOrientedDrive`